pep-0682.rst

```diff
@@ -113,11 +105,11 @@ one-dimensional numerical arrays would be complicated by such pre-rounding:
         return f"[{','.join(f'{term:{format_spec}}' for term in v)}]"
 
 To date, there doesn't appear to be other widely-used languages or libraries
```

Comment:
Not something that needs to be fixed in this PR, but the grammar doesn't seem quite right here (agreement between 'doesn't' and the plural 'languages or libraries'). Maybe:

> To date, there do not appear to be other [...]

or perhaps

> To date, there does not appear to be any other widely-used language or library [...]